typescript/alert-center/src/operations.ts

```diff
@@ -0,0 +1,106 @@
+import { WorkflowContext, Workflow, PostApi, HandlerContext, ArgOptional, configureInstance, GetApi } from '@dbos-inc/dbos-sdk';
+import { RespondUtilities, AlertEmployee, AlertStatus, AlertWithMessage, Employee } from './utilities';
+import { Kafka, KafkaConfig, KafkaProduceCommunicator, Partitioners, KafkaConsume, KafkaMessage, logLevel } from '@dbos-inc/dbos-kafkajs';
+export { Frontend } from './frontend';
+import { CurrentTimeCommunicator } from "@dbos-inc/communicator-datetime";
+
+const respondTopic = 'alert-responder-topic';
+
+const kafkaConfig: KafkaConfig = {
+  clientId: 'dbos-kafka-test',
+  brokers: [`${process.env['KAFKA_BROKER'] ?? 'localhost:9092'}`],
+  logLevel: logLevel.ERROR
+};
+
+const producerConfig: KafkaProduceCommunicator =  configureInstance(KafkaProduceCommunicator, 
+  'wfKafka', kafkaConfig, respondTopic, {
+    createPartitioner: Partitioners.DefaultPartitioner
+  });
+
+const timeToPackAlert = 30;
+
+export interface AlertEmployeeInfo
+{
+  employee: Employee;
+  alert: AlertEmployee[];
+  expirationSecs: number | null;
+  newAssignment: boolean;
+}
+
+@Kafka(kafkaConfig)
+export class Respondment {
+  @Workflow()
+  @KafkaConsume(respondTopic)
+  static async inboundAlertWorkflow(ctxt: WorkflowContext, topic: string, _partition: number, message: KafkaMessage) {
+    if (topic !== respondTopic) return; // Error
```

Comment:
Do we need this check here? It shouldn't happen because DBOS guarantees the topic to match.

Reply:
Should be addressed